steps/jira-tickets/0.0.1/step.yml

```diff
@@ -0,0 +1,71 @@
+title: JIRA Tickets listing
+summary: |
+  List all JIRA tickets from comparing two branches
+description: |
+  List all JIRA tickets from comparing two branches
+website: https://bitbucket.org/silofit/bitrise-jira-tickets/src/master/
+source_code_url: https://bitbucket.org/silofit/bitrise-jira-tickets/src/master/
+support_url: https://bitbucket.org/silofit/bitrise-jira-tickets/src/master/
+published_at: 2021-07-29T11:13:33.163625-04:00
+source:
+  git: https://amarliac@bitbucket.org/silofit/bitrise-jira-tickets.git
```

Comment:
Step id should be rather list-jira-tickets or find-jira-tickets.
jira-tickets itself is ambiguous.